docs/user-guide/scaling.rst

```diff
@@ -249,22 +249,24 @@ Load Balancing Requirements
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 To effectively handle inbound HTTP/HTTPS requests to `Pulp clustered servers`
-running ``httpd``, a load balancer should be used. Configuring a load
-balancer is beyond the scope of the Pulp documentation, but there are a few
-recommendations.
+running ``httpd``, load balancing of some sort should be used. Configuring
+load balancing is beyond the scope of Pulp documentation, but there are a
+few recommendations.
 
-Pulp defaults to using SSL for webserver traffic so the easiest thing is to
-use a TCP based load balancer. `HAProxy`_ has been tested with a clustered
-Pulp installation, but any TCP load balancer should work.
+One option is to use a dedicated load balancer. Pulp defaults to using SSL
+for webserver traffic, so an easy thing is to use a TCP based load
+balancer. `HAProxy`_ has been tested with a clustered Pulp installation,
+but any TCP load balancer should work.
 
-With TCP load balancing, all `Pulp clustered servers` need to have ``httpd``
-configured with the same certificate. That certificate needs to have the CN
-set to the hostname or IP of the load balancer. This configuration ensures
-that the load balancer passes traffic to the Pulp webservers, but that the
-client and server will negotiate a SSL connection directly. By setting the CN
-to the hostname or IP of the load balancer, the client will trust the
-certificate presented by the `Pulp clustered server` as it is accessed
-through the load balancer.
+Another option is to use DNS based load balancing. Community users have
+reported this works, but it has not been explicitly tested by Pulp
+developers.
+
+With either load balancing technique, all `Pulp clustered servers` need to
```

Comment:
Perhaps clarify that this only applies to machines they want to run httpd on. Maybe it's implied, but I wouldn't want someone to misinterpret this as saying they're required to run httpd on every machine.